haiku-kata/src/test/java/org/eclipse/collections/haikukata/TextProcessorECTest.java

```diff
@@ -27,6 +27,7 @@
 public class TextProcessorECTest
 {
     @Test
+    @Tag("KATA")
```

Comment:
It seems the builds are failing as it is not able to find the class "Tag". You are missing import in this class. 
`import org.junit.jupiter.api.Tag;`

Same comment on the 4 files.

Reply:
Thank you so much for your guidance @prathasirisha , I will update it.